src/data/Data.js

```diff
@@ -577,9 +577,10 @@ export const WorkspaceData = _.flow(
     const region = isAzureWorkspace ? storageDetails.azureContainerRegion : storageDetails.googleBucketLocation;
 
     const wdsDataTableProvider = useMemo(() => {
-      const proxyUrl = !!wdsProxyUrl && wdsProxyUrl.state;
+      const app = !!wdsApp && wdsApp.state;
```

Comment:
[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) would probably be a better fit here.

With optional chaining, `app` would be `App | undefined` vs `App | false` and `proxyUrl` `string | undefined` vs `string | false`.

Reply:
Nice!   I am not a fan of mixed types if I can avoid them and forgot all about optional chaining :)